addon/utils/query-utils.js

```diff
@@ -2,6 +2,8 @@ import $ from 'jquery';
 
 const { trim } = $;
 
+// TODO: allow-list instead of block-list?
```

Comment:
This should go into an issue instead of a todo. It would require some discussion since I'm not sure we actually can whitelist.

Reply:
OK, removed.